### PR TITLE
Ben first branch

### PR DESCRIPTION
**Pull request by benparks1976**
I'd like someone to review this before the changes are merged

Please note the conflict junk that needs to be fixed.

### CODE REVIEW

**Comment by rvanrijssel** [comment]
Very nice

**Comment by robinvanderhoeven** [comment]
Reviewed, ok.

File: new-file.txt [hunk 1]
```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
```
Comment (BenParks32) [inline new-file.txt]: fix this

**Review by BenParks32 — CHANGES_REQUESTED** [review]
Do better!

**Review by BenParks32 — APPROVED** [review]

**Pull request merged**